### PR TITLE
Various change to visitor doc

### PR DESCRIPTION
**Pull request by antham**
Fix a typo and doc comment.

### CODE REVIEW

File: visitor/dumper.go [hunk 1]
```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/z7zmey/php-parser/walker"
 )
 
-// Dumper prints ast hierarchy to stdout
```
Comment (z7zmey) [inline visitor/dumper.go]: It is correct to change `stdout` to `io.Writer`.

Dumper is expected any struct that implements io.Writer interface not only stdout.

**Comment by z7zmey** [comment]
Thank you for help. The documentation is the hardest part for me, especially in English.

File: visitor/dumper.go (hunk 1 above)
Comment (antham) [reply visitor/dumper.go]: You want me to replace Writer with io.Writer ?

Comment (z7zmey) [reply visitor/dumper.go]: Yep

Comment (antham) [reply visitor/dumper.go]: Done, if you want to have a look

**Pull request merged**